src/Microsoft.Health.Fhir.SqlServer.Api/Features/LatestSchemaVersionHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Messages.Get;
+
+namespace Microsoft.Health.Fhir.SqlServer.Api.Features
+{
+    public class LatestSchemaVersionHandler : IRequestHandler<GetCompatibilityVersionRequest, GetCompatibilityVersionResponse>
```

Comment:
Would be good to have the handler name similar to the name of the request that it handles to improve readability. Maybe something like `GetCompatibleSchemaVersionHandler` or something like that?